tools/upload_text.php

```diff
@@ -345,6 +343,10 @@
         }
     }
 
+    if ($stage == 'smooth_done') {
```

Comment:
Lets put opening `{` on a newline so the format matches the rest (or most) of this file.